helm/ph-ee-engine/values.yaml

```diff
@@ -2105,3 +2105,163 @@ vouchers:
     hostname: "https://ops-bk.sandbox.mifos.io"
     endpoints:
       transfers: "/api/v1/transfers?size=1&page=0"
+
+billPay:
+  enabled: false
+  replicas: 1
+  image: docker.io/openmf/ph-ee-connector-bill-pay:v1.0.0
+  imagePullPolicy: "Always"
+  livenessProbe:
```

Comment:
duplicate lines
same are added to service level values.yaml as well
this can be skipped here